include/deal.II/sundials/arkode.h

```diff
@@ -1373,19 +1374,12 @@ namespace SUNDIALS
     /**
      * Constructor.
      *
-     * @param solver The ARKode solver that uses this operator
      * @param A_data Data required by @p a_times_fn
      * @param a_times_fn A function pointer to the function that computes A*v
      */
-    SundialsOperator(ARKode<VectorType> &solver,
-                     void *              A_data,
-                     ATimesFn            a_times_fn);
+    SundialsOperator(void *A_data, ATimesFn a_times_fn);
```

Comment:
Doesn't this break backward-compatibiliy?

Reply:
This looks like a new function (`#if DEAL_II_SUNDIALS_VERSION_GTE(4, 0, 0)`) so it is safe to change.

Reply:
Correct. In addition, the whole construct will hopefully be removed before release and replaced with the LinearOperator.